### ISSUE TITLE
Some buttons don't work

### ISSUE THREAD

**Issue by griffi-gh** [deadlyjack/Acode#481]
Arrow and tab buttons don't work (but surprisingly Shift+Tab does work just fine)

**Comment by deadlyjack** [comment]
Hi, have you tried new updates if it fixed your issue or not?

**Comment by MagTun** [comment]
I have the same problem. At first it worked but after an update, several months ago it stopped working. I was still on Android 10 and I assumed that the rom was just "too old" but I have recently migrated to Lineage 20 (Android 13) and I still have the same problem. I was using Acode from Play store, but I have just reinstalled it from F-droid  (1.8.5) and I still have the same problem. I use Hacker keyboard with has inbuilt arrow key on the keyboard and they also do not work. 



![image](https://github.com/deadlyjack/Acode/assets/15843700/ae5e31a6-14af-47a4-8197-564ee01c8831)
